### PR TITLE
Changed List.of to Arrays.asList and changed the settings to jdk 8.

### PR DESCRIPTION
**Pull request by talbz**
Changed the file to be under /tmp.

### CODE REVIEW

File: src/main/java/org/pokemon/PokemonApp.java [hunk 1]
```diff
@@ -18,14 +18,14 @@ public static void main(String[] args) {
         HighestAttackHandler highestAttackHandler = new HighestAttackHandler();
         PokemonStreamAcceptor pokemonStreamAcceptor = new PokemonStreamAcceptor();
         PokemonStreamHandler pokemonStreamHandler = new PokemonStreamHandler(
-                List.of(maxHpHandler,
+                Arrays.asList(maxHpHandler,
```
Comment (tomer-ben-david) [inline src/main/java/org/pokemon/PokemonApp.java:21]: Is there a specific reason you prefer Arrays.asList.   I just looked at the difference I prefer List.of - immutable, does not allow nulls – more strict.  But List.of is only there from Java9, is that the reason?

File: src/main/java/org/pokemon/dal/PokemonFileDAO.java [hunk 2]
```diff
@@ -41,4 +41,23 @@ public List<Pokemon> getAll() {
         }
         return pokemons;
     }
+
+    private Pokemon buildPokemon(String line) {
```
Comment (tomer-ben-david) [inline src/main/java/org/pokemon/dal/PokemonFileDAO.java:45]: Better indeed. thanks looks more organized got your structure.

File: src/main/java/org/pokemon/PokemonApp.java (hunk 1 above)
Comment (tomer-ben-david) [reply src/main/java/org/pokemon/PokemonApp.java:21]: I now see your branch is named java 8 :) and also the description - so already answered : )

Comment (talbz) [reply src/main/java/org/pokemon/PokemonApp.java:21]: About the immutability, you are right. I think in Java 8, we can use guava for it. Or do this
List<String> stringList = Arrays.asList("a", "b", "c");
stringList = Collections.unmodifiableList(stringList);

**Pull request merged**